### ISSUE TITLE
Setup docs/examples need to cover setting `--default-volume-snapshot-locations`

### ISSUE THREAD

**Issue by skriss** [heptio/ark#994]
Admins need to modify their deployment YAML to specify the name(s) of the default VSLs that should be used for backups if they want users to not have to explicitly specify them every time a backup is taken.

We need to cover this topic in our setup docs - and we should probably add the flag to our example deployment YAML as well, either will a <YOUR_VSLS> placeholder, or commented out with inline documentation.

**Comment by skriss** [comment]
@Bradamant3 we're working through a code change that should make addressing this issue less necessary, since specifying this flag will no longer be required in the normal case.  It would probably still be good to document it somewhere, but I'm not exactly sure where ATM (agree that our examples right now are not exhaustive, but illustrative).

**Comment by Bradamant3** [comment]
I've got a temporary solution worked out locally -- should be fine with or without code change. The whole install flow needs an overhaul, but not in the 0.10 timeframe.

**Comment by skriss** [comment]
Not quite.  During the install, in the deployment YAML, you'd add an arg to the container cmd:
```
--default-volume-snapshot-locations aws:us-east-1,portworx:local,...
```

**Comment by Bradamant3** [comment]
Fixed at least for now with PR #1008
